docs/reference/transform/painless-examples.asciidoc

```diff
@@ -18,9 +18,16 @@ more about the Painless scripting language in the
 * <<painless-compare>>
 * <<painless-web-session>>
 
-NOTE: While the context of the following examples is the {transform} use case, 
+[NOTE] 
+--
+* While the context of the following examples is the {transform} use case, 
 the Painless scripts in the snippets below can be used in other {es} search 
 aggregations, too.
+* {transforms-cap} rely on dynamic mappings when creating the destination index. 
+They cannot deduce mappings of output fields when the fields are created by a 
+script. In this case, create the mappings of the destination index prior to 
+creating the {transform}. 
```

Comment:
- I like to emphasize that the limitation only applies to the script examples on this page, not in general
- "rely on dynamic mappings when creating the destination index" - we simply do not generate any mapping, dynamic mapping of the fields is what ES ootb, this is not controlled by transform
- "create the mappings of the destination index prior to creating the transform", you can create the transform, dest index creation is done at start, therefore this should be "prior to _starting_ the transform"

Suggestion:

All the following examples use scripts, transform can not deduce mappings of output fields when the fields are created by a script. Transform will not create any mappings in the the destination index for these fields, which means they get dynamically mapped. You can create the destination index prior to starting the transform in case you want explicit mappings.

Reply:
@hendrikmuhs Thank you for the input. I addressed your comment via 456d92f.